maintainers/maintainer-list.nix

```diff
@@ -9834,4 +9834,10 @@
     github = "hloeffler";
     githubId = 6627191;
   };
+  preisschild = {
```

Comment:
Can you put this into a separate commit per Contribution Guide?

Reply:
Done. Thanks for the information, must have read over it.